controllers/chat/sendAnonymousMessage.js

```diff
@@ -67,6 +68,8 @@ const sendAnonymousMessage = async (req, res) => {
     if (createdChannel?.channel?.is_channel_blocked)
       return res.status(403).json(responseError('Channel is blocked'));
 
+    let channelMember = [];
+    let senderInfo = [];
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

These variables `channelMember` and `senderInfo` are declared as arrays but later on they are assigned objects (lines 86-90). This could lead to confusion about their intended use and data type. Consider initializing them as `null` or directly where they are assigned.
```diff
- let channelMember = [];
- let senderInfo = [];
+ let channelMember = null;
+ let senderInfo = null;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->